src/context/build-utils.js

```diff
@@ -0,0 +1,12 @@
+const errorMessages = require('../utils/error-messages');
+const errorUtils = require('../utils/error');
+const stringUtils = require('../utils/string');
+const joinUrl = require('../utils/join-url');
+const { setFieldWidget } = require('../utils/widgets');
```

Comment:
[peanuts]
the requires should be in the factory function (in order to not require anything in case of mock)